src/app/LandingPage/components/Slogan/Slogan.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Banner = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 0px;
+    position: absolute;
+    width: 722px;
+    height: 128px;
+    left: 270px;
+    top: 224px;
+`;
+
+const Heading = styled.div`
+    text-align: left;
+    font-size: 28px;
+    letter-spacing: 8px;
+    font-size: 32px;
+    line-height: 64px;
+    letter-spacing: 8px;
+    font-feature-settings: 'liga' off;
```

Comment:
这个是不是从 figma 上抄下来的？不需要
```suggestion
    letter-spacing: 8px;
```

Reply:
已改